goodkey/good_key.go

```diff
@@ -237,15 +237,24 @@ func (policy *KeyPolicy) goodKeyRSA(key *rsa.PublicKey) (err error) {
 	if modulusBitLen%8 != 0 {
 		return berrors.MalformedError("key length wasn't a multiple of 8: %d", modulusBitLen)
 	}
-	// The CA SHALL confirm that the value of the public exponent is an
-	// odd number equal to 3 or more. Additionally, the public exponent
-	// SHOULD be in the range between 2^16 + 1 and 2^256-1.
-	// NOTE: rsa.PublicKey cannot represent an exponent part greater than
-	// 2^32 - 1 or 2^64 - 1, because it stores E as an integer. So we
-	// don't need to check the upper bound.
-	if (key.E%2) == 0 || key.E < ((1<<16)+1) {
-		return berrors.MalformedError("key exponent should be odd and >2^16: %d", key.E)
+
+	// Rather than support arbitrary exponents, which significantly increases
+	// the size of the key space we allow, we restrict E to the defacto standard
+	// RSA exponent 65537. There is no specific standards document that specifies
+	// 65537 as the 'best' exponent, but ITU X.509 Annex C suggests there are
+	// notable merits for using it if using a fixed exponent.
+	//
+	// The CABF Baseline Requirements state:
+	//   The CA SHALL confirm that the value of the public exponent is an
+	//   odd number equal to 3 or more. Additionally, the public exponent
+	//   SHOULD be in the range between 2^16 + 1 and 2^256-1.
+	//
+	// By only allowing one exponent that fits these constraints we satisfy
```

Comment:
Nitpick: This sentence could also be read as "We allow many exponents, only one of which fits these constraints."

```suggestion
	// By only allowing one exponent, which fits these constraints, we satisfy
```